pkg/flavor/kubernetes/flavor.go

```diff
@@ -130,6 +130,11 @@ func (flavor *Flavor) LoadNodeInfo(node *model.Node) (string, error) {
 	log.Tracef(">>>>> LoadNodeInfo called with node %v", node)
 	defer log.Trace("<<<<< LoadNodeInfo")
 
+	// overwrite actual host name with node name from k8s to be compliant
+	if name := os.Getenv("NODE_NAME"); name != "" {
```

Comment:
would need to update the yaml/helm file to set this env, right ?

Reply:
yes..

Reply:
cool. if for some reason it's not set should we default to existing hostname or that is already taken care of?

Reply:
yes Raunak, if its set to empty, we will default to original behavior.